bootstrap.sh

```diff
@@ -89,6 +89,7 @@ else
 
   if [ `uname -s` == "Darwin" ]; then
     # on OSX tox is installed in the following path
+    export PYTHONPATH=$(prepend_path $PYTHONPATH $grpc_dist/usr/local/lib/python2.7/site-packages)
```

Comment:
After https://github.com/youtube/vitess/pull/1541 "tox" should no longer be required and this path shouldn't be on $PYTHONPATH

Can you please check the content of the site-packages directory on your system? It should be empty when you do a fresh bootstrap (delete $grpc_dist completely).